src/main/webapp/intro.html

```diff
@@ -0,0 +1,286 @@
+        <li class="nav-item">
+          <a class="nav-link" href="index.html">Home</a>
+        </li>
+        <li class="nav-item">
+          <!-- TODO: Create new html page about the project  -->
+          <a class="nav-link" href="intro.html">About</a>
+        </li>  
+      </ul> 
+        <!-- Team -->
+      <section id="team" class="pb-5">
+        <div class="container">
+            <h5 class="section-title h1">Team NAN</h5>
+            <div class="row">
+                <!-- Team member -->
+                <div class="col-xs-12 col-sm-6 col-md-4">
+                    <div class="image-flip" >
+                        <div class="mainflip flip-0">
+                            <div class="frontside">
+                                <div class="card">
+                                    <div class="card-body text-center">
+                                        <p><img class=" img-fluid" src="images/Molly.jpg" alt="card image"></p>
+                                        <h4 class="card-title">Molly Graton</h4>
+                                        <p class="card-text">Descibe Molly in a sentence or less. IAmAPlaceholder IAmAPlaceholder.</p>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="backside">
+                                <div class="card">
+                                    <div class="card-body text-center mt-4">
+                                        <h4 class="card-title">Molly</h4>
+                                        <p class="card-text">Molly is a rising sophomore at University of Illinois Urbana-Champaign studying computer science. </p>
+                                        <ul class="list-inline">
+                                          <li class="list-inline-item">
+                                            <a class="social-icon text-xs-center" target="_blank" href="https://www.linkedin.com/in/mollygraton/">
+                                                <i class="fa fa-google"></i>
+                                            </a>
+                                          </li>
+                                            <li class="list-inline-item">
+                                                <a class="social-icon text-xs-center" target="_blank" href="https://www.linkedin.com/in/mollygraton/">
+                                                    <i class="fa fa-facebook"></i>
+                                                </a>
+                                            </li>
+                                            <li class="list-inline-item">
+                                                <a class="social-icon text-xs-center" target="_blank" href="https://www.linkedin.com/in/mollygraton/">
+                                                    <i class="fa fa-twitter"></i>
+                                                </a>
```

Comment:
Should we fill in all the info before merging?

Reply:
yes~ all up to y'all~ i just added the button for it right now